features/post-meta.feature

```diff
@@ -219,6 +219,39 @@ Feature: Manage post custom fields
       My\New\Meta
       """
 
+Scenario: List post meta with or without single flag
+    Given a WP install
+
+    When I run `wp post meta add 1 apple banana`
+    And I run `wp post meta add 1 apple banana`
```

Comment:
This `And` line seems redundant.

Reply:
Should we use `And I run the previous command again`?

Reply:
For testing purposes here I would add multiple different meta values, not just banana